package.json

```diff
@@ -70,7 +70,7 @@
     "jquery-i18next": "1.2.1",
     "js-md5": "0.6.1",
     "jwt-decode": "2.2.0",
-    "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#9e5d83f4aceb9f922583c871d933e97ee2b05753",
+    "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#2be643757510a3d122330b8898858f99d099841f",
```

Comment:
Don't update LJM as part of a PR, use the update-ljm.sh script to create a dependency bump PR.